spec/rails_helper.rb

```diff
@@ -0,0 +1,69 @@
+require 'simplecov'
```

Comment:
added the `rails_helper` file required at the beginning of each test file

Reply:
Good idea, thank you